### PR TITLE
Extend open-tracing to include misformed PSK indentities.

### PR DESCRIPTION
**Pull request by boaks**
Signed-off-by: Achim Kraus <achim.kraus@bosch.io>

### CODE REVIEW

File: adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/PreSharedKeyDeviceIdentity.java [hunk 1]
```diff
@@ -20,6 +20,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import io.opentracing.Span;
```
Comment (sophokles73) [inline adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/PreSharedKeyDeviceIdentity.java]: can you please adjust the change period in the copyright header as well?

File: adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/DefaultDeviceResolver.java [hunk 2]
```diff
@@ -132,7 +132,7 @@ public AdditionalInfo getInfo(final Principal clientIdentity) {
 
         if (clientIdentity instanceof PreSharedKeyIdentity) {
             final Span span = newSpan("PSK-getDeviceIdentityInfo");
-            final PreSharedKeyDeviceIdentity deviceIdentity = getHandshakeIdentity(clientIdentity.getName());
+            final PreSharedKeyDeviceIdentity deviceIdentity = getHandshakeIdentity(span, clientIdentity.getName());
```
Comment (sophokles73) [inline adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/DefaultDeviceResolver.java]: can you please adjust the change period in the copyright header as well?

Comment (sophokles73) [reply adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/DefaultDeviceResolver.java]: The period is specified as `LOWER, UPPER`, i.e. `2019, 2020` ...

**Comment by boaks** [comment]
Adapted to
2019, 2020
2018, 2020

(I haven't changed the second file in 2019, so I hope 2018, 2020 is right.)

File: adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/DefaultDeviceResolver.java (hunk 2 above)
Comment (boaks) [reply adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/DefaultDeviceResolver.java]: Adapted to
2019, 2020
2018, 2020

**Pull request merged**